py-polars/src/polars/series/string.py

```diff
@@ -2050,6 +2051,10 @@ def replace_many(
             Enable ASCII-aware case-insensitive matching.
             When this option is enabled, searching will be performed without respect
             to case for ASCII letters (a-z and A-Z) only.
+        leftmost
+            Guarantees in case there are overlapping matches that the leftmost match
+            is used. In case there are multiple candidates for the leftmost match
+            the pattern which comes first in patterns is used.
```

Comment:
I think `replace_many` on `Series` should also have the new examples.

Reply:
oh indeed, fixed